### ISSUE TITLE
[designspaces] weight axis values need to be mapped from 1-1000

### ISSUE THREAD

**Issue by m4rc1e** [TypeNetwork/Roboto#12]
According to [Microsoft Weight Axis tag spec](https://docs.microsoft.com/en-us/typography/opentype/spec/dvaraxistag_wght)

> Valid numeric range: Values must be in the range 1 to 1000.

We're inheriting the values from the [old build process](https://github.com/google/roboto/blob/master/scripts/build-v2.py#L87-L112) which have a range of -1-1.125

Unfortunately these values are not css friendly (font-weight is broke)

![screen shot 2018-06-27 at 16 19 05](https://user-images.githubusercontent.com/7525512/41983381-dadbd42c-7a25-11e8-80d7-b8c937414a36.png)
*Setting ccs font-weight does not match previous family instances*

To fix this, we update the weight axis in each designspace file to.

```
        <axis default="525.294118" maximum="900" minimum="250" name="weight" tag="wght">
            <labelname xml:lang="en">Weight</labelname>
            <map input="250" output="-1" />
            <map input="525.294118" output="-0.1" />
            <map input="900" output="1.125" />
        </axis>
```

We're now assigning a map input to each output val.

We should now get a better result

![screen shot 2018-06-27 at 16 30 22](https://user-images.githubusercontent.com/7525512/41984000-712bcfa8-7a27-11e8-8009-1a3ed2b5a6a8.png)
*Appearance is still off because fonts lack avar table*

To get a 100% match, I need to work out why the avar tables are not being generated.

@cjdunn I wouldn't worry about this. I'll update update it myself once I get a working avar table as well.

**Comment by davelab6** [comment]
@m4rc1e this will be resolved when #13 is merged, correct?

**Comment by m4rc1e** [comment]
Yes
